src/plugins/unified_search/public/filter_bar/filter_badges_wrapper.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { EuiFlexItem } from '@elastic/eui';
+import { InjectedIntl, injectI18n } from '@kbn/i18n-react';
+import type { Filter } from '@kbn/es-query';
+import React, { useRef } from 'react';
+
+import { METRIC_TYPE } from '@kbn/analytics';
+import { FilterItem } from './filter_item';
+import { useKibana } from '../../../kibana_react/public';
+import { IDataPluginServices } from '../../../data/public';
+import { DataView } from '../../../data_views/public';
+
+export interface Props {
+  filters: Filter[];
+  onFiltersUpdated?: (filters: Filter[]) => void;
+  indexPatterns: DataView[];
+  intl: InjectedIntl;
+  appName: string;
```

Comment:
I found one thing here: `appName` is being required as a prop, but doesn't actually get used as it ends up getting `appName` from `useKibana`

Reply:
Correct!  Fixed https://github.com/elastic/kibana/pull/129877/commits/406b9008571acb0b492529187ddaaafad43aef1a